scheduler_test.go

```diff
@@ -881,3 +881,29 @@ func TestDo(t *testing.T) {
 		})
 	}
 }
+
+func TestRemoveAfterExec(t *testing.T) {
+	f := func(s *Scheduler) {
+		fmt.Print("i'm a dummy task")
+	}
+
+	s := NewScheduler(time.UTC)
+	s.StartAsync()
+
+	job1, err := s.Every(1).StartAt(time.Now().Add(10*time.Second)).Do(f, s)
+	require.NoError(t, err)
+
+	job1.LimitRunsTo(1)
+	job1.RemoveAfterLastExec()
+
+	time.Sleep(15 * time.Second)
+
+	var jobExists bool
+	for _, j := range s.jobs {
+		if j == job1 {
+			jobExists = true
+		}
+	}
+
+	assert.False(t, jobExists)
```

Comment:
since you're only adding the one job here, you could just assert the len of `s.Jobs()` is 0
```suggestion
	assert.Zero(t, len(s.Jobs()))
```